src/ngraph/runtime/cpu/pass/cpu_fusion.hpp

```diff
@@ -72,6 +72,11 @@ class ngraph::runtime::cpu::pass::CPUFusion : public ngraph::pass::GraphRewrite
             construct_conv_bias();
             construct_sigmoid_multiply();
         }
+        if (fusions & REGULAR_FUSIONS)
+        {
+            construct_conv_add();
```

Comment:
Add to the previous conditional for REGULAR_FUSIONS

Reply:
If added to previous condition, then conv_bias was getting replaced with conv_add. 
Hence, I added it after conv_bias.

Reply:
Oh, right. We should be able to move all the differentiable_fusions before regular_fusions. Would be good to add a comment about the ordering requirement between conv_bias and conv_add.

Reply:
Yes, moving all differentiable_fusions before regular_fusions works. Will add comment too.